less/index.less

```diff
@@ -54,6 +54,155 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	min-height: 100%;
 }
 
+html {
+	font-size: 62.5%;
+	font-family: 'Montserrat', sans-serif;
+}
+
+// variables
+
+// mix-ins
+
+.section-heading-content() {
+	h1 {
+		font-size: 4.2rem;
+		color: firebrick;
+		font-weight: bold;
+	}
+
+	h2 {
+		font-size: 2.0rem;
+		font-weight: bold;
+		margin: 25px 0 8px 0;
+	}
+
+	h3 {
+		font-size: 1.4rem;
+		margin: 8px 0;
+	}
+
+	p {
+		font-size: 1.4rem;
+		margin: 20px 0;
+	}
+}
+
+// fixed header
+
+header {
+	display: flex;
+	align-items: center;
+	justify-content: flex-end;
+	position: fixed;
+	background-color: black;
+	width: 100%;
+	height: 70px;
+	top: 0;
+	left: 0;
+	z-index: 1;
+	box-shadow: 0 0 30px 0;
+
+	nav {
+		display: flex;
+		justify-content: space-around;
+		width: 30%;
+
+		a {
+			color: white;
+			font-size: 1.8rem;
+			text-decoration: none;
+		}
+
+		a:hover {
+			text-decoration: underline;
+		}
+	}
+}
+
+main {
+	display: flex;
+	height: 100%;
+
+	.side-container {
+		width: 100%;
+		height: 100%;
+		max-width: 350px;
+		background-color: firebrick;
+
```

Comment:
`min-height: 100%` on your `main` container and on your `.side-container` fixes your 100% height issue on the side-container.